crud/src/main/java/org/openmbee/sdvc/crud/services/NodePostHelper.java

```diff
@@ -66,20 +66,20 @@ public NodeChangeInfo processPostJson(List<ElementJson> elements, boolean overwr
             if (element.getId() == null || element.getId().isEmpty()) {
                 element.setId(UUID.randomUUID().toString());
             }
-            ElementJson elasticElement = info.getExistingElementMap().get(element.getId());
+            ElementJson indexElement = info.getExistingElementMap().get(element.getId());
             Node n = info.getExistingNodeMap().get(element.getId());
             if (n == null) {
                 added = true;
-            } else if (elasticElement == null) {
+            } else if (indexElement == null) {
```

Comment:
can you also change the 'elastic' word in the log message to index? thx

Reply:
updated